src/main/java/org/ilgcc/app/utils/SubmissionUtilities.java

```diff
@@ -314,4 +315,35 @@ public static Map<String, Object> getProviderScheduleByRepeatForValue(Map<String
      List<Map<String, Object>> providerSchedules = (List<Map<String, Object>>) childcareSchedule.getOrDefault("providerSchedules", emptyList());
      return providerSchedules.stream().filter(providerSchedule -> providerSchedule.get("repeatForValue").equals(repeatForValue)).toList().stream().findFirst().orElse(null);
   }
+
+  public static void respondForCurrentProvider(Submission providerSubmission, Submission familySubmission) {
+    String currentProviderUuid = providerSubmission.getInputData().get("currentProviderUuid").toString();
+    String providerResponseAgreeToCare = (String) providerSubmission.getInputData().get("providerResponseAgreeToCare");
+
+    List<Map<String, Object>> providers = getProviders(familySubmission.getInputData());
+    boolean allProvidersResponded = true;
+
+    for (int i = 0; i < providers.size(); i++) {
```

Comment:
Small nit: could this be written like this? 
```suggestion
   for(Map<String, Object> provider : providers){
```

Reply:
no, because Marc uses the index in line 332: providers.set(i, provider);

Reply:
Yep, good call. It should work without the `i` -- just pushed a change!

Reply:
I just learned something today :)